dll/win32/shell32/lang/pt-PT.rc

```diff
@@ -256,7 +256,7 @@ BEGIN
     LTEXT "Atributos:", 14020, 8, 152, 46, 10
     AUTOCHECKBOX "&Só de leitura", 14021, 58, 151, 68, 10
     AUTOCHECKBOX "&Oculto", 14022, 126, 151, 55, 10
-    AUTOCHECKBOX "&Arquivo", 14023, 181, 151, 59, 10
+    AUTOCHECKBOX "A&rquivar", 14023, 181, 151, 59, 10
```

Comment:
Naive question: why the change of keyboard accelerator here?

Reply:
Just following in the footsteps of Win2003?